<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CampTutorial</class>
 <widget class="qMRMLWidget" name="CampTutorial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>347</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input markups:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputMarkupSelector">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsFiducialNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>true</bool>
        </property>
        <property name="removeEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkSliderWidget" name="opacitySliderWidget">
        <property name="toolTip">
         <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="pageStep">
         <double>0.100000000000000</double>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoUpdateCheckBox">
     <property name="text">
      <string>Auto-update</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Output</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output folder: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="outputLineEdit">
        <property name="filters">
         <set>ctkPathLineEdit::Dirs|ctkPathLineEdit::Drives|ctkPathLineEdit::Executable|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::Readable</set>
        </property>
        <property name="options">
         <set>ctkPathLineEdit::ShowDirsOnly</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="exportDataButton">
        <property name="text">
         <string>Export data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>CampTutorial</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputMarkupSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
